hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java

```diff
@@ -799,14 +798,13 @@ private boolean deleteBucket(Path f, boolean recursive, OFSPath ofsPath)
     // if so, the contents of bucket were deleted and skip delete bucket
     // otherwise, Handle delete bucket
     if (!handleTrailingSlash) {
-      deleteBucketFromVolume(f, ofsPath);
+      deleteBucketFromVolume(f, bucket);
     }
     return result;
   }
 
-  private boolean isLinkBucket(Path f, OFSPath ofsPath) {
+  private boolean isLinkBucket(Path f, OzoneBucket bucket) {
     try {
```

Comment:
Exception handling is no longer needed.  With that, this method can be inlined (`bucket.isLink()` used directly).

Reply:
Thank you @adoroszlai for the review, made the suggested changes!